src/modules/export.js

```diff
@@ -407,7 +407,7 @@ export const startExport = () => {
 
       let formatter;
       if (exportData.fileType === 'csv') {
-        formatter = createCSVFormatter();
+        formatter = createCSVFormatter({ columns: Object.keys(projection) });
```

Comment:
Nice trick!
At the moment the projection truncates nested fields to depth=2,
like for: {a.b.c.d} --> 'a.b', will this keep those?

Reply:
Good point, I only tested up until depth 2 so far – thanks for catching this!

Reply:
Okay, I’ve updated this PR quite a bit … please take another look :)